src/Sulu/Bundle/MediaBundle/Resources/public/js/services/image-editor.js

```diff
@@ -12,9 +12,16 @@ define(['config', 'jquery'], function(Config, $) {
     'use strict';
 
     var featherEditor;
+    var featherUrl;
 
     if (!!Config.get('sulu-media').adobeCreativeKey) {
-        require(['//feather.aviary.com/imaging/v3/editor.js'], function() {
+        if (location.protocol === 'https:') {
```

Comment:
Is this necessary? I think there is no reason not to use HTTPS variant when we are running on HTTP.  Only the other way is a problem.